### PR TITLE
Fix Protocol Form Descriptions And Typos

### PR DESCRIPTION
**Pull request by yasinmiran**
## Purpose
Please note $subject

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -730,6 +726,13 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     showPredictions={ false }
                                     customLabel={ allowedOriginsErrorLabel }
                                 />
+                                <Hint>
+                                    The HTTP origins that host your web application. You can define multiple web
```
Comment (ayshsandu) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:730]: @yasinmiran Shouldn't these be coming from I18n file?

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -814,6 +814,20 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                         readOnly={ readOnly }
                         data-testid={ `${ testId }-access-token-type-radio-group` }
                     />
+                    <Message compact={false} size={"tiny"} className={"border-less"}>
+                        <p>
+                            Asgardeo has the capability to attach the OAuth2 access token and refresh
```
Comment (ayshsandu) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:819]: In the product "WSO2 Identity Server", and these should come from i18n file.

Comment (yasinmiran) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:819]: We need to take i18n to extensions as well. So we can override specific messages like these. Will create an issue for this

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (yasinmiran) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:730]: Yes, it must refer from i18n/console. I will create an issue for this.

**Pull request merged**